polaris.shopify.com/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import GlobalSearch from "../GlobalSearch";
+import MaxPageWidthDiv from "../MaxPageWidthDiv";
+import Button from "../Button";
+import SideMenu from "../SideMenu";
+import NavItems from "../NavItems";
+
+import shopifyLogo from "../../../public/shopify-logo.svg";
+import hamburguerIcon from "../../../public/images/icon-hamburguer.svg";
+import styles from "./Header.module.scss";
+
+interface Props {
+  currentSection?: string;
+}
+
+function Header({ currentSection }: Props) {
+  const [showMenu, setShowMenu] = useState(false);
+
+  useEffect(() => {
+    // it ensures that the menu is closed after the screen is resized
+    function handleResize() {
+      if (window.innerWidth > 768 && showMenu) {
```

Comment:
Can we extract `768` into a variable in a config file?

Reply:
Uuhmmm I thought about creating an enum and keep it at `types.ts`
At least until we have come utility functions that deal with resolution and breakpoints

what do you think?